spec/feature/restaurant_feature_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'restaurant'
+
+describe Restaurant do
+  subject(:restaurant){described_class.new}
+
+  xdescribe '#show_menu' do
+
+    it 'outputs the list of menu items' do
+      expect{restaurant.show_menu}.to output(Restaurant::MENU.each {|key, value| "#{key} = #{value}"}).to_stdout
```

Comment:
Metrics/LineLength: Line is too long. [112/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)